Romina.Api/Repositories/ProductRepository.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Data.SqlClient;
+using System.Linq;
+using Dapper;
+using Romina.Api.Models;
+
+namespace Romina.Api.Repositories
+{
+    public class ProductRepository: IProductRepository
+    {
+        public Product GetProductById(string id)
+        {
+            Product product = null;
+            using (SqlConnection connection = new SqlConnection("localConnectionString"))
```

Comment:
I see `localConnectionString` is a placeholder, but how would we pass in the connection string from config? 

see here for an example of how to do this => https://weblog.west-wind.com/posts/2016/may/23/strongly-typed-configuration-settings-in-aspnet-core

Reply:
hey bud we still aren't passing in a "settings" object (we require an object that contains the sql connection string). 

This repository is still stuck on using `localConnectionString`. Look at the blog post I've linked above again and have a look what he did in the ConfigurationController

Reply:
the above comment still stands ^

Reply:
hey bud the comments above still stands